dist/screeps.d.ts

```diff
@@ -1017,8 +1017,7 @@ interface SignDefinition {
 }
 interface StoreDefinition {
     [resource: string]: number;
-    energy?: number;
-    power?: number;
+    energy: number;
```

Comment:
The resources type would have to be a mapping of name to value, unless you can transform an array to it, but yes, I can definitely support such a change. Great suggestion @ChrisKitching!